<!DOCTYPE html>
<html lang="en-us">
  <head>


    <meta charset="UTF-8">
    <title>HHV6-GLUE by giffordlabcvr</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="website/stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="website/stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="website/stylesheets/github-light.css" media="screen">


  </head>
  
  <body>
  
    <section class="page-header">
      <h1 class="project-name">HHV6-GLUE</h1>
      <h2 class="project-tagline">A GLUE project for human herpesvirus 6 (HHV6)</h2>
      <a href="viruses.html" class="btn">Viruses</a>
      <a href="paleoviruses.html" class="btn">Paleoviruses</a>
      <a href="https://github.com/giffordlabcvr/HHV6-GLUE/zipball/master" class="btn">Download</a>
      <a href="https://github.com/giffordlabcvr/HHV6-GLUE" class="btn">View on GitHub</a>
    </section>

    <section class="main-content">




	   <h3>
		 <a id="overview" class="anchor" href="#overview" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Overview</strong>
	   </h3>
         <hr>

		 <p>
		   This is <b>CRESS-GLUE</b>, an open access
		   <a href="http://glue-tools.cvr.gla.ac.uk/"><b>GLUE</b></a>
		   project designed to support comparative genomic and evolutionary analysis of
		   circular Rep-encoding single-stranded DNA
		   (<a target="_blank" href="https://www.sciencedirect.com/science/article/pii/S0065352718300575"><b>CRESS DNA</b></a>)
		   viruses (phylum 
		   <a target="_blank" href="https://jvi.asm.org/content/94/12/e00582-20"><b><i>Cressdnaviricota</i></b>)</a>. 
		 </p>

		 <br>
        
         <p><img src="./website/images/cress-glue-header.jpg" alt="CRESS DNA virus phylogeny" /></p>
         
         
		 <br>

<blockquote>
		 <p>
		   The figure above, taken from a report by
		   <a target="_blank" href="https://jvi.asm.org/content/94/12/e00582-20"><b>Krupovic <i>et al.</i> (2020)</b></a>,	    
		   shows an 
		   <a target="_blank" href="https://evolution-outreach.biomedcentral.com/articles/10.1007/s12052-010-0273-6"><b>unrooted phylogenetic tree</b></a>
		   of the <b><i>Cressdnaviricota</i></b>,
		   based on alignment of	   
		   <a target="_blank" href="https://www.uniprot.org/uniprot/Q8BB16"><b>Rep</b></a>
		   proteins.
		 </p>      
</blockquote>
		 
  



	  <h3>
		<a id="contributors" class="anchor" href="#contributors" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Contributors</strong>
	  </h3>

	  <p>Robert J. Gifford (<a href="mailto:robert.gifford@glasgow.ac.uk">robert.gifford@glasgow.ac.uk</a>)</p>


       
		 <br>

		 <p>
		 
		   <b>CRESS-GLUE</b>
		   has been designed to facilitate any form of comparative genomic analysis
		   involving <b>CRESS DNA viruses</b>. It contains a richly annotated sequence
		   dataset for these viruses, comprised of both 
		   <a href="./website/project.html"><b>viral sequences</b></a>
		   and
		   <a href="./website/paleovirology.html"><b>endogenous viral elements (EVEs)</b></a>.
		   
		 </p>


		 <p>
		 
		 
          <b>There are a wide variety of ways in which this resource can be used</b>:
           
		 <ul>
		   <li> if you're interested in any particular virus in the <i>Cressdnaviricota</i>,
		   you can use CRESS-GLUE as an efficient means of investigating this virus in depth,
		   as  
		   <a href="./website/project-uses-example.html"><b>demonstrated in this tutorial</b></a>.

		   </li><br>
		   
		   <li> for those interested in <b>paleovirology</b> and <b>endogenous viral elements (EVEs)</b>,
		   CRESS-GLUE provides a source of systematically organised information about
		   <a href="./website/paleovirology.html"><b>EVEs derived from CRESS DNA viruses</b></a>.
		   </li><br>

		   <li> if you're interested in exploring <b>CRESS DNA virus diversity</b> 
		   in <b>metagenomic</b> datasets, the data included in this project 
		   may assist you in your analysis, as explained
		   <a href="./website/paleovirology.html"><b>here</b></a>.
		   </li><br>
		   
		 </ul>
		  
		 </p>



	   <h3>
	   <a id="WhatGlueProject" class="anchor" href="#glueProject" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>What is a GLUE project?</strong>
	   </h3>
       <hr>


	   <p>
		 <a target="_blank" href="http://glue-tools.cvr.gla.ac.uk/"><b>GLUE</b></a> is an open, integrated
		 software toolkit that provides functionality for storage and interpretation of
		 sequence data. 
	   </p>

	   <p>
		 <b>GLUE</b> supports the development of <b>“projects”</b> containing the data items
		 required for comparative genomic analysis 
		 (e.g.  sequences, multiple sequence alignments, genome feature annotations,
		 and other sequence-associated data). 
	   </p>

	   <p>
		 Projects are loaded into the <b>GLUE "engine"</b>, creating a relational database
		 that represents the semantic relationships between data items.
		 This provides a robust foundation for the implementation of <b>systematic
		 comparative analyses</b> and the development of <b>sequence-based resources</b>. 
	   </p>

	   <p>		 
		  The core schema of this database can be extended to accommodate
		  the idiosyncrasies of different projects, and
		  GLUE provides a scripting layer (based on JavaScript) for developing custom
		  analysis tools. 
	   </p>

	   <p>
		  Hosting of GLUE projects in an online version control system (e.g. GitHub) provides
		  a mechanism for their stable, collaborative development.
	   </p>
	   
	   

		   
		 <p>
		   Some examples of '<b>sequence-based resources</b>' built for viruses using GLUE include:

		 </p>
		   
		   

		 <p>

		 <ul>
		 		 
		   <li> 
			 <a target="_blank" href="http://cov-glue.cvr.gla.ac.uk/"><b>COV-GLUE</b></a>: 
			 A GLUE resource for tracking genetic variation in SARS-COV2.
			 CoV-GLUE contains a database of amino acid replacements, insertions and
			 deletions which have been observed in GISAID hCoV-19 sequences sampled from the pandemic
		   </li>
		   <br>
		   
		   <li>
			 <a target="_blank" href="http://rabv-glue.cvr.gla.ac.uk/"><b>RABV-GLUE</b></a>:
			 Tailored toward epidemiological tracking of rabies virus (RABV).
			 Includes a database of RABV sequences and metadata from NCBI, updated daily and arranged into major and minor clades, and
			 an analysis tool providing genotyping, analysis and visualisation of submitted FASTA sequences.
 		   </li>
 		   <br>
		   
		   <li>
			 <a target="_blank" href="http://hcv-glue.cvr.gla.ac.uk/"><b>HCV-GLUE</b></a>:
			 This GLUE resource aims to support analysis of drug resistance and vaccine
			 escape in hepatitis C virus (HCV).
			 A database of HCV sequences and metadata from NCBI, updated daily and arranged
			 into clades (genotypes, subtypes). As well as pre-built multiple-sequence
			 alignments of NCBI sequences, it includes an analysis tool providing genotyping,
			 drug resistance analysis and visualisation of submitted FASTA sequences.
           </li>
	   
		 </ul>

		   
		 </p>

	   
       <br>
	   <h3>
	   <a id="WhyGlueProject" class="anchor" href="#glueProject" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>What does building the CRESS-GLUE project offer?</strong>
	   </h3>
       <hr>

 
	   <p>
		 
		  CRESS-GLUE offers a number of advantages for performing comparative sequence
		  analysis of CRESS DNA viruses:

		 <ol>
		   <li> <strong>Standardisation of sequence orientation</strong>. Like many 
		   viruses with circular genomes, the orientation of circovirus sequences 
		   is not standardised in GenBank. CRESS-GLUE addresses this issue.
		   </li>
		   <li> <strong>Standardisation of the genomic co-ordinate space</strong>. GLUE
		   projects allow all sequences to utilise the coordinate space of a chosen 
		   reference sequence. Contingencies associated with insertions and deletions
		   (indels) are handled in a systematic way.
		   </li>
		   <li> <strong>Predefined, fully annotated reference sequences</strong>:
		   This project includes fully-annotated reference sequences for major lineages
		   within the CRESS DNA assemblage.
		   </li>
		   <li> <strong>Alignment trees</strong>:
		   GLUE allows linking of alignments constructed at distinct taxonomic levels
		   via an "alignment tree" data structure. In the alignment tree, each alignment
		   is constrained to a standard reference sequence, thus all multiple sequence
		   alignments are linked to one another via a standardised coordinate system.

		   </li>
		 </ol>

	   </p>

         <br>



	   <h3>
	   <a id="glueProject" class="anchor" href="#glueProject" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Building this GLUE project</strong>
	   </h3>
         <hr>


      	   
	   <p>
	
         On computers with the GLUE software framework
         <a target="_blank" href="http://glue-tools.cvr.gla.ac.uk/#/installation"><b>installed</b></a>,
         the <b>CRESS-GLUE</b> project can be instantiated by navigating to the project folder,
         initiating GLUE, and issuing the following command in the GLUE shell:
         
  <pre>
  <code>Mode path: /
  GLUE> run file buildCoreProject.glue</code></pre>

         This will build the CRESS-GLUE core project by executing the commands in
		 <a href="https://github.com/giffordlabcvr/CRESS-GLUE/blob/master/cressCoreProject.glue"><b>this file</b></a>.
		 
	   </p>

	   <p>
		 The core project comprises a  
		 dataset designed to represent the phylum in a minimal way (i.e. by including 
		 only one or a handful of annotated reference sequences for each major lineage). 
	   </p>


	   <p>
		 We have also created extension projects for certain families in the 
		 <b><i>Cressdnaviricota</i></b> (e.g. <i>Circoviridae</i>).
		 These extensions extend the minimal set of
		 reference sequences included in the core project to include representatives of
		 taxonomic groups below family-level.
	   </p>


	   <p>
		 Once the core project has been built, the family-level extension project can be
		 added by executing the commands in
		 <a href="https://github.com/giffordlabcvr/CRESS-GLUE/blob/master/cressCircoviridaeProject.glue"><b>this file</b></a>,
		 as follows:
         
  <pre>
  <code>Mode path: /
  GLUE> run file buildFamilyLevelProjects.glue</code></pre>
		 		  
	   </p>


	   <p>
		 The family-level extension projects can be extended to incorporate
		 EVE sequences derived from the corresponding virus family.
		 To build these paleovirus extensions, execute the commands in
		 <a href="https://github.com/giffordlabcvr/CRESS-GLUE/blob/master/cressPaleoExtensionCircoviridae.glue"><b>this file</b></a>,
		 as follows:
         
  <pre>
  <code>Mode path: /
  GLUE> run file buildFamilyLevelPaleoProjects.glue</code></pre>
		 		  
	   </p>


	   <p>
		 We have also created family-level, extension project for certain families in the 
		 <b><i>Cressdnaviricota</i></b>.
	   </p>


	   <p>
		 The <i>Circoviridae</i> paleovirus extension incorporates a set of endogenous
		 viral elements (EVEs) recovered from the genomes of metazoan species. Building
		 the paleovirus extension allows automated alignment and phylogeny reconstruction
		 for individual ECV lineages in the project, based on the classifications
		 in
		 <a target="blank" href="https://github.com/giffordlabcvr/CRESS-GLUE/blob/master/tabular/eve/circoviridae/ecv-circo-side-data.tsv"><b>this file</b></a>.
		 Individual ECV sequences have been classified into sets considered likely to 
		 have arisen from the same germline colonisation event. Loci have been named using
		 a systematic approach. 
		 
	   </p>
       <br>


	   <h3>
		 <a id="contributors" class="anchor" href="#contributors" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Contributors</strong>
	   </h3>
         <hr>
	   
	   <p>
		  <a target="_blank" href="https://giffordlabcvr.github.io/virus-evolution/index.html">Robert J. Gifford</a> (<a href="mailto:robert.gifford@glasgow.ac.uk">robert.gifford@glasgow.ac.uk</a>)	 
	   </p>
	   <p>
          Tristan Dennis
	   </p>
	   <p>
          Soledad Marsile-Medun
	   </p>

	   <br>


	   <h3>
		 <a id="Publications" class="anchor" href="#Publications" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Related Publications</strong>
	   </h3>
         <hr>

         Krupovic M, Varsani A, Kazlauskas D, Breitbart M, Delwart E, Rosario K, Yutin N,
         Wolf YI, Harrach B, Zerbini FM, Dolja VV, Kuhn JH, and EVE Koonin.
		<b>(2020)</b>
		<br>
		Cressdnaviricota: a Virus Phylum Unifying Seven Families of Rep-Encoding Viruses with Single-Stranded, Circular DNA Genomes.
		<br>
		<i>J. Virol.</i>. Jun 1;94(12):e00582-20. doi: 10.1128/JVI.00582-20.
		<a href="https://jvi.asm.org/content/94/12/e00582-20" target="_blank">[view]</a> 
		<br>
		<br>



        Tisza MJ, Pastrana DV, Welch NL, Stewart B, Peretti A, Starrett GJ, Pang YS,
        Krishnamurthy SR, Pesavento PA, McDermott DH, Murphy PM, Whited JL, Miller B,
        Brenchley J, Rosshart SP, Rehermann B, Doorbar J, Ta'ala BA, Pletnikova O,
        Troncoso JC, Resnick SM, Bolduc B, Sullivan MB, Varsani A, Segall AM, Buck CB.
		<b>(2020)</b>
		<br>
		Discovery of several thousand highly diverse circular DNA viruses.
		<br>
		<i>Elife</i>. Feb 4;9:e51971. doi: 10.7554/eLife.51971.
		<a href="https://elifesciences.org/articles/51971" target="_blank">[view]</a> 
		<br>
		<br>



		Singer JB, Thomson EC, McLauchlan J, Hughes J, and RJ Gifford
		<b>(2018)</b>
		<br>
		GLUE: A flexible software system for virus sequence data.
		<br>
		<i>BMC Bioinformatics</i>
		<a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2459-9" target="_blank">[view]</a> 
		<br>
		<br>



		Singer JB, Gifford RJ, Cotten M, and D Robertson
		<b>(2020)</b>
		<br>
		CoV-GLUE: A Web Application for Tracking SARS-CoV-2 Genomic Variation.
		<br>
		<i>Preprints</i>
		<a href="https://www.preprints.org/manuscript/202006.0225/v1" target="_blank">[preprint]</a> 
		<br>
		<br>

 		</p>
 		
        <br>



	   <h3>
		 <a id="license" class="anchor" href="#license" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>License</strong>
	   </h3>
       <hr>

		 <p>
		   This project is licensed under the <a href="https://www.gnu.org/licenses/agpl-3.0.en.html">GNU Affero General Public License v. 3.0</a>.
		 </p>

	   <br>



	  <footer class="site-footer">
		<span class="site-footer-owner"><a href="https://github.com/giffordlabcvr/HHV6-GLUE">HHV6-GLUE</a> is maintained by <a href="https://github.com/giffordlabcvr">giffordlabcvr</a>.</span>

		<span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
	  </footer>

    </section>


  </body>
</html>
