<!DOCTYPE html>
<html lang="en-us">
  <head>


    <meta charset="UTF-8">
    <title>HHV6-GLUE by giffordlabcvr</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="website/stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="website/stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="website/stylesheets/github-light.css" media="screen">


  </head>
  
  <body>
  
    <section class="page-header">
      <h1 class="project-name">HHV6-GLUE</h1>
      <h2 class="project-tagline">A GLUE project for human herpesvirus 6 (HHV6)</h2>
      <a href="./website/viruses.html" class="btn">Viruses</a>
      <!-- <a href="./website/paleoviruses.html" class="btn">Paleoviruses</a> -->
      <a href="https://github.com/giffordlabcvr/HHV6-GLUE/zipball/master" class="btn">Download</a>
      <a href="https://github.com/giffordlabcvr/HHV6-GLUE" class="btn">View on GitHub</a>
    </section>

    <section class="main-content">


	  <h3>
	  <a id="background" class="anchor" href="#background" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Background/Description</strong>
	  </h3>


	  <p>
		  <b>Human Herpesvirus 6 (HHV-6)</b> comprises two closely related betaherpesviruses (genus
		  <a target="_blank"  href="https://viralzone.expasy.org/16"><i>Betaherpesvirus</i></a>)
		  called <b>HHV-6A</b> and <b>HHV-6B</b>. 
		  Like other herpesviruses, HHV-6 establishes persistent, chronic infections.
		  Infection is usually <b>latent</b>, but <b>reactivation</b> can sometimes occur
		  and may lead to disease symptoms. 
	  </p>

       
      <p><img src="./website/images/hhv6-header.jpg" alt="CRESS DNA virus phylogeny" /></p>
         

	  <blockquote>
		  <b>Human herpesviruses</b>. Left to right:
		  (i) Paired chromosomes, with telomeric regions indicated in green:
		  uniquely among human herpesviruses, HHV-6A and HHV-6B
		  can establish life-long latency via integration into the subtelomeric regions
		  of human chromosomes;
		  (ii) Cryo-EM structure of a human herpesvirus capsid 
		  (<a target="_blank"  href="https://doi.org/10.1126/science.aao7283">HHV2</a>);
		  (iii) Histological slide showing HHV6 infected cells, with
		  <a target="https://www.sciencedirect.com/topics/immunology-and-microbiology/inclusion-bodies">inclusion bodies</a>
		  in both the nucleus and the cytoplasm. 
	      (iv) HHV-6 reactivation is relatively common during pregnancy,
	      and in approximately 1% of pregnancies HHV-6 may be transferred to the fetus.
	  </blockquote>


	  <p>
		  HHV6 was first identified in 1986. Many aspects of its biology and its 
		  impact on human health remain to be understood. Remarkably, around 1-2% of people
		  carry HHV6-derived sequences in their chomosomal DNA as alleles. These
		  '<b>chromosomally-integrated HHV6 (ciHHV6)</b>' sequences represent a type of 
		  <b>endogenous viral element (EVE)</b> - a genomic DNA sequence that derives from
		  an ancestral virus.
          Recent studies show that chromosomally integrated HHV-6 can <b>excise</b>
          from chromosomes and activate under certain circumstances (e.g. immune
          deficiency, following transplantation).
	  </p>



	  <p>
		  "Except in acute or initial infections, the viral DNA can
		  typically be found only by biopsy, as it does not circulate in peripheral blood.
		  There is an urgent need for new technologies to detect these occult infections that
		  are not apparent by standard quantitative PCR DNA testing methods, in order to prove
		  or disprove the important disease associations that have been suggested." -- 
		  <a target="_blank"  href="https://hhv-6foundation.org/"><b>HHV6 Foundation</b></a>
	  </p>
 
 
	   <h3>
	   <a id="WhatGlueProject" class="anchor" href="#glueProject" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>What is a GLUE project?</strong>
	   </h3>
       <hr>


	   <p>
		 <a target="_blank" href="http://glue-tools.cvr.gla.ac.uk/"><b>GLUE</b></a> is an open, integrated
		 software toolkit that provides functionality for storage and interpretation of
		 sequence data. 
	   </p>

	   <p>
		 <b>GLUE</b> supports the development of <b>“projects”</b> containing the data items
		 required for comparative genomic analysis 
		 (e.g.  sequences, multiple sequence alignments, genome feature annotations,
		 and other sequence-associated data). 
	   </p>

	   <p>
		 Projects are loaded into the <b>GLUE "engine"</b>, creating a relational database
		 that represents the semantic relationships between data items.
		 This provides a robust foundation for the implementation of <b>systematic
		 comparative analyses</b> and the development of <b>sequence-based resources</b>. 
	   </p>

	   <p>		 
		  The core schema of this database can be extended to accommodate
		  the idiosyncrasies of different projects, and
		  GLUE provides a scripting layer (based on JavaScript) for developing custom
		  analysis tools. 
	   </p>

	   <p>
		  Hosting of GLUE projects in an online version control system (e.g. GitHub) provides
		  a mechanism for their stable, collaborative development.
	   </p>
	   
	   

		 
	   <p>
		 Some examples of '<b>sequence-based resources</b>' built for viruses using GLUE include:

	   </p>
		 
	   

	   <p>

	   <ul>
			   
		 <li> 
		   <a target="_blank" href="http://cov-glue.cvr.gla.ac.uk/"><b>COV-GLUE</b></a>: 
		   A GLUE resource for tracking genetic variation in SARS-COV2.
		   CoV-GLUE contains a database of amino acid replacements, insertions and
		   deletions which have been observed in GISAID CoV-19 sequences sampled from the pandemic
		 </li>
		 <br>
		 
		 <li>
		   <a target="_blank" href="http://rabv-glue.cvr.gla.ac.uk/"><b>RABV-GLUE</b></a>:
		   Tailored toward epidemiological tracking of rabies virus (RABV).
		   Includes a database of RABV sequences and metadata from NCBI, updated daily and arranged into major and minor clades, and
		   an analysis tool providing genotyping, analysis and visualisation of submitted FASTA sequences.
		 </li>
		 <br>
		 
		 <li>
		   <a target="_blank" href="http://hcv-glue.cvr.gla.ac.uk/"><b>HCV-GLUE</b></a>:
		   This GLUE resource aims to support analysis of drug resistance and vaccine
		   escape in hepatitis C virus (HCV).
		   A database of HCV sequences and metadata from NCBI, updated daily and arranged
		   into clades (genotypes, subtypes). As well as pre-built multiple-sequence
		   alignments of NCBI sequences, it includes an analysis tool providing genotyping,
		   drug resistance analysis and visualisation of submitted FASTA sequences.
		 </li>
	 
	   </ul>

		 
	   </p>
	 



	   <br>
	   <h3>
	   <a id="WhyGlueProject" class="anchor" href="#glueProject" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>What does building the HHV6-GLUE project offer?</strong>
	   </h3>
	   <hr>

		 
 
	   <p>
		 
		  HHV6-GLUE offers a number of advantages for performing comparative sequence
		  analysis of HHV6A and HHV6B:


		  <ol>

			<li> <strong>Reproducibility</strong>. 
			For many reasons, bioinformatics analyses are notoriously difficult to reproduce.
			The GLUE framework supports the implementation of fully reproducible 
			comparative genomics through the introduction of data standards and the use 
			of a relational database to capture the semantic links between data items.
			</li>
			<br>
		  
			<li> <strong>Reusable data objects and analysis logic</strong>. 
			For many - if not most - comparative genomic analyses, data preparation is nine
			tenths of the battle. The GLUE framework has been designed to ensure that 
			work spent preparing high-value data items such as multiple sequence alignments
			need only be performed once. Hosting of GLUE projects in an online version control 
			system such as GitHub allows for collaborative management of important data items
			and community testing of hypotheses.
			</li>
			<br>


			<li> <strong>Validation</strong>. 
			Building GLUE projects entails mapping the semantic links between data items
			(e.g. sequences, tabular data, multiple sequence alignments).
			This process provides an opportunity 
			for cross-validation, and thereby enforces a high level of data integrity.
			</li>
			<br>
		
			<li> <strong>Standardisation of the genomic co-ordinate space</strong>. GLUE
			projects allow all sequences to utilise the coordinate space of a chosen 
			reference sequence. Contingencies associated with insertions and deletions
			(indels) are handled in a systematic way.
			</li>
			<br>
		  
			<li> <strong>Predefined, fully annotated reference sequences</strong>:
			This project includes fully-annotated reference sequences for major lineages
			within the <i>Hepadnaviridae</i> family.
			</li>
			<br>
		  
			<li> <strong>Alignment trees</strong>:
			GLUE allows linking of alignments constructed at distinct taxonomic levels
			via an ""<b>alignment tree</b>"
			data structure. In the alignment tree, each alignment
			is constrained to a standard reference sequence, thus all multiple sequence
			alignments are linked to one another via a standardised coordinate system.
			</li>
			
		  </ol>


	   </p>



	   <h3>
		 <a id="Publications" class="anchor" href="#Publications" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Related Publications</strong>
	   </h3>


		 <p>
	   
	       Aswad A, Aimola G, Wight D, Roychoudhury P, Zimmermann C, Hill J, Lassner D, Xie H, Huang ML,
	       Parrish NF, Schultheiss HP, Venturini C, Lager S, Smith GCS, Charnock-Jones DS, Breuer J, Greninger AL, and BB Kaufer.
		   <b>(2021)</b>
		   <br>
		   Evolutionary History of Endogenous Human Herpesvirus 6 Reflects Human Migration out of Africa.
		   <br>
		   <i>BMC Mol Biol Evol.</i>
		   <a href="https://doi.org/10.1093/molbev/msaa190" target="_blank">[view]</a> 
		   <br>
		   <br>



           Liu X, Kosugi S, Koide R, Kawamura Y, Ito J, Miura H, Matoba N, Matsuzaki M,
           Fujita M, Kamada AJ, Nakagawa H, Tamiya G, Matsuda K, Murakami Y, Kubo M,
           Aswad A, Sato K, Momozawa Y, Ohashi J, Terao C, Yoshikawa T, Parrish NF, and Y Kamatani.
		   <b>(2020)</b>
		   <br>
		   Endogenization and excision of human herpesvirus 6 in human genomes.
		   <br>
		   <i>PLoS Genet.</i>
		   <a href="https://doi.org/10.1371/journal.pgen.1008915" target="_blank">[view]</a> 
		   <br>
		   <br>

   
		   Singer JB, Thomson EC, McLauchlan J, Hughes J, and RJ Gifford
		   <b>(2018)</b>
		   <br>
		   GLUE: A flexible software system for virus sequence data.
		   <br>
		   <i>BMC Bioinformatics</i>
		   <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2459-9" target="_blank">[view]</a> 
		   <br>
		   <br>



           Ablashi D, Agut H, Alvarez-Lafuente R, Clark DA, Dewhurst S, DiLuca D, Flamand L,
           Frenkel N, Gallo R, Gompels UA, Höllsberg P, Jacobson S, Luppi M, Lusso P, Malnati M,
           Medveczky P, Mori Y, Pellett PE, Pritchett JC, Yamanishi K, and T Yoshikawa.
		   <b>(2014)</b>
		   <br>
		   Classification of HHV-6A and HHV-6B as distinct viruses.
		   <br>
		   <i>Arch Virol. </i>		   

		   <a href="https://doi.org/10.1007/s00705-013-1902-5" target="_blank">[view]</a> 
		   <br>
		   <br>

 
		   Katzourakis A. and RJ. Gifford
		   <b>(2010)</b>
		   <br>
		   Endogenous viral elements in animal genomes.
		   <br>
		   <i>PLoS Genetics</i>
		   <a href="http://www.plosgenetics.org/article/info:doi/10.1371/journal.pgen.1001191" target="_blank">[view]</a>

		 </p>



	   <h3>
		 <a id="contributors" class="anchor" href="#contributors" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Contributors</strong>
	   </h3>

		 <p>Robert J. Gifford (<a href="mailto:robert.gifford@glasgow.ac.uk">robert.gifford@glasgow.ac.uk</a>)</p>



	   <h3>
		 <a id="license" class="anchor" href="#license" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>License</strong>
	   </h3>
       <hr>

		 <p>
		   This project is licensed under the <a href="https://www.gnu.org/licenses/agpl-3.0.en.html">GNU Affero General Public License v. 3.0</a>.
		 </p>

	   <br>





	  <footer class="site-footer">
		<span class="site-footer-owner"><a href="https://github.com/giffordlabcvr/HHV6-GLUE">HHV6-GLUE</a> is maintained by <a href="https://github.com/giffordlabcvr">giffordlabcvr</a>.</span>

		<span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
	  </footer>

    </section>


  </body>
</html>
