<!DOCTYPE html>
<html lang="en-us">

  <head>

    <meta charset="UTF-8">
    <title>HHV6-GLUE by giffordlabcvr: Paleovirology components</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="stylesheets/github-light.css" media="screen">

  </head>

  <body>
    <section class="page-header">
      <h1 class="project-name">HHV6-GLUE</h1>
      
      <h2 class="project-tagline">A GLUE project for the HHV6.</h2>
      <a href="../index.html" class="btn">Home</a>
      <a href="" class="btn">Paleoviruses</a>
 
      <a href="https://github.com/giffordlabcvr/HHV6-GLUE/zipball/master" class="btn">Download</a>
      <a href="https://github.com/giffordlabcvr/HHV6-GLUE/tree/master" class="btn">View on GitHub</a>
      
    </section>

    <section class="main-content">

	   <h3>
		 <a id="Background" class="anchor" href="#Background" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Endogenous herpesvirus (ciHHV6) data</strong>
	   </h3>

		 <p>

		   Whole genome sequencing has revealed the presence of DNA sequences derived from
		   herpesviruses in vertebrate genomes. These <b>‘chromosomally integrated HHV6’ elements
		   (ciHHV6s)</b> are thought to have originated via ‘germline incorporation’ events in
		   which herpesvirus DNA sequences were integrated into chromosomal DNA of germline
		   cells and subsequently inherited as novel host alleles. 
		 </p>


		 <p>

		   ciHHV6 sequences are in some ways
		   equivalent to herpesvirus ‘fossils’ in that they provide a source of retrospective
		   information about the distant ancestors of modern herpesviruses. 
		 </p>


         <p><img src="./images/herpesvirus-header.png" alt="Timeline of herpesvirus evolution" /></p>



<blockquote>
           <b>Timeline of HHV6 evolution</b>. 
</blockquote>
		 

	   <br>




	   <p>
         Reflecting this, analysis of ciHHV6s has proven immensely informative with respect to the long-term
         evolutionary history of herpesviruses. 
         For example, in a recent study
         (Lytras <i>et al</i>, 2020),
         we uncovered evidence that the 
         genus <i>Aviherpesvirus</i> comprises at least four distinct subgroups, each
         of which has circulated widely among avian species at some point during the 
         Cenozoic Era. Each subgroup is characterised by a distinct type of Surface protein.
 	   </p>


	   <h3>
		 <a id="Virology" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Relevance to molecular biological studies of herpesviruses</strong>
	   </h3>        
	   <hr>

	   <p>
          Endogenous viral sequences can inform our understanding of contemporary viruses
          in a wide variety of ways. Perhaps most importantly, EVEs allow calibration of
          the long-term evolutionary history of virus groups, which greatly influences 
          how we understand their biology.
 	   </p>

	   <p>
         Importantly, once time calibrations have been established, a far richer range of
         comparative genomic studies can be performed. By examining
         variation in the light of a known evolutionary history, these studies can provide 
         invaluable insights into the biological mechanisms through which viruses 
         replicate and spread.
	   </p>



        <p><img src="./images/herpes-paleovirus-header.png" alt="Herpesvirus EVEs" /></p>


<blockquote>
		<p>
		  Some of the species in which we have identified <b>EVEs derived from herpesviruses</b> 
		  (commonly referred to as <b>endogenous hepatitis B (ciHHV6) elements</b>)
		  <br>
		  From left to right:
		  emperor penguin (<i>Aptenodytes forsteri</i>), barn owl (<i>Tyto alba</i>), cormorants (family Phalacrocoracidae), Anna's hummingbird (<i>Calypte anna</i>).
		</p>
         
</blockquote>
		 


     
	   <br>
	   <h3>
		 <a id="Metagenomics" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Relevance to viral metagenomics</strong>
	   </h3>        
	   <hr>

	   <p>
          The ciHHV6 sequences in HHV6-GLUE can
          provide a useful resource for those
          interested in identifying and characterising herpesviruses in metagenomic
          datasets.
	   </p>


	   <p>
          Firstly, the ciHHV6 sequences collated here can be used to exclude any potential
          'false positive' hits (i.e. sequences that seem to represent new herpesviruses but in fact derive
          from genomic DNA).
	   </p>


	   <p>
          In addition, when new herpesvirus species are identified, inclusion of EVEs
          in phylogenetic analyses can often provide useful information
          about their broader ecology and evolution, including (uniquely) their long-term
          evolution. 
	   </p>
       

       
	   <br>
	   <h3>
		 <a id="Genomics" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Relevance to genomics</strong>
	   </h3>        
	   <hr>

	   <p>
          ciHHV6s are not only useful genetic markers, several lines of evidence indicate
          they may have, or have had, functional roles as host alleles. The prevalence of 
          multicopy ciHHV6 lineages in some species suggests that germline incorporation of
          herpesvirus sequences might have influenced the evolution of host genomes in
          important ways.
	   </p>

	   <p>

          Consistent with the idea that germline incorporation of herpesvirus
          sequences might in some cases be favoured by selection at the level of the host,
          we have identified several examples of loci containing multiple fixed ciHHV6 elements,
          each derived from a distinct germline colonisation event. It remains unclear
          whether this reflects natural selection due to a favourable influence or
          preferential integration of herpesviruses into particular loci (e.g. because they
          are accessible in embryonic cells).      

	   </p>


        <p><img src="./images/herpes-paleovirus-midlevel.png" alt="Herpesvirus EVEs" /></p>


<blockquote>
		<p>
		  Additional species in which we have identified <b>endogenous hepatitis B (ciHHV6) elements</b>
		  <br>
		  From left to right:
		  tuatara (<i>Sphenodon punctatus</i>), ducks (family Anatidae), snakes (suborder Serpentes), red-legged seriema (<i>Cariama cristata</i>).
		</p>
         
</blockquote>
		 



 
 	   <br>
	   <h3>
		 <a id="EveProject" class="anchor" href="#EveProject" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>The EVE component of HHV6-GLUE</strong>
	   </h3>
	   <hr>

	   
		<p>

          Currently, the distribution and diversity of HHV6-related sequences
          in animal genomes remains incompletely characterized.
          Progress in characterising these elements has been hampered by the challenges
          encountered attempting to analyse their fragmentary and degenerated sequences. 
          <b>HHV6-GLUE</b> aims to address these issues.
		</p>

	   
		<p>
          We have incorporated into this project a set of principles for
          organising the herpesvirus 'fossil record', and a protocol 
          through which it can be accessed and collaboratively developed.
		</p>

        <p>
          This website provides background information about the EVE component of 
          <b>HHV6-GLUE</b>, and direct links to specific data items. 

		</p>

 	   <br>
	   <h3>
		 <a id="Nomenclature" class="anchor" href="#Nomenclature" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Nomenclature for ciHHV6s</strong>
	   </h3>
	   <hr>

		<p>
          We have applied a systematic approach to naming ciHHV6, following a convention 
          developed for <a target="_blank" href="https://retrovirology.biomedcentral.com/articles/10.1186/s12977-018-0442-1">endogenous retroviruses</a>.
		  Each element was assigned a unique identifier (ID) constructed from a defined
		  set of components.
  		</p>


 
        <p><img src="./images/nomenclature-ciHHV6.png" alt="ciHHV6 Nomenclature" /></p>

 		


		<p>
          The first component is the classifier ‘ciHHV6’ (endogenous hepatitis B virus/endogenous herpesvirus). 
		</p>

		<p>
          The second component is a composite of two distinct subcomponents separated by a period:
          
          (i)  the name of ciHHV6 group;
          (ii) a numeric ID that uniquely identifies the insertion.
          The numeric ID is an integer that identifies a unique insertion locus that arose as a
          consequence of an initial germline infection. Thus, orthologous copies in different
          species are given the same number. 
 		</p>
		
		<p>
          Where an EVE sequence is thought to have been duplicated within the germline
          following it's initial incorporation (e.g. via segmental duplication or transposition)
          we have appended an additional 'duplicate id' to the numeric ID, separated by a period.
          Please note that we have not yet resolved the orthologous relationships among
          sets of ciHHV6 sequences belonging to multicopy ciHHV6 lineages. We
          have therefore assigned unique duplicate IDs to each sequence within these lineages. 
		</p>
		
		<p>
          The third component of the ID defines the set of host species in which
          the ortholog occurs, or did occur prior to being deleted.
  		</p>

 
	   <br>
	   <h3>
		 <a id="Raw ciHHV6 sequences and data" class="anchor" href="#Raw_digs_data" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Raw ciHHV6 sequences and data</strong>
	   </h3>
	   <hr>

		<p>
          These are the raw data generated by
          <a target="_blank" href="https://giffordlabcvr.github.io/DIGS-tool/">database-integrated genome screening (DIGS)</a>.
          The 
          <a target="_blank" href="https://github.com/giffordlabcvr/HHV6-GLUE/blob/master/tabular/eve/ehbv-side-data.tsv"><strong>tabular file</strong></a>
          contains information about the genomic location of each EVE. 
          EVEs were classified by comparison to a
          <a href="https://github.com/giffordlabcvr/HHV6-GLUE/blob/master/digs/herpes_refs_consolidate_final.faa">reference library of herpesvirus polypeptide sequences</a>
          library designed to represent the known diversity of herpesviruses - this includes
          extinct lineages represented only by endogenous viral elements (EVEs).
 		</p>

		<p>
          These data were obtained via
          <a target="_blank" href="https://giffordlabcvr.github.io/DIGS-tool/">DIGS</a>
          performed in vertebrate genome assemblies downloaded from 
          <a href="https://www.ncbi.nlm.nih.gov/genome/browse ">NCBI genomes</a>
          (2020-07-15).
		</p>


		<p>
           Raw data about the EVEs in tabular format can be found <a target="_blank" href="https://github.com/giffordlabcvr/HHV6-GLUE/blob/master/tabular/eve/ehbv-side-data.tsv"><strong>here</strong></a>.
  		</p>

		<p>
           Nucleotide level data in FASTA format (individual files) can be found <a target="_blank" href="https://github.com/giffordlabcvr/HHV6-GLUE/tree/master/sources/fasta-digs-ehbv"><strong>here</strong></a>.
  		</p>
		



	   <br>
	   <h3>
		 <a id="Phylogenetic trees" class="anchor" href="#PhylogeneticTrees" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Phylogenetic trees</strong>
	   </h3>
	   <hr>

		<p>
          We used GLUE to implement an automated process for deriving midpoint rooted,
          annotated trees from the alignments included in our project.
 		</p>


		<p>
          Trees were constructed at distinct taxonomic levels:
        
		  <ol>
			<li> Recursively populated <a href="https://github.com/giffordlabcvr/HHV6-GLUE/tree/master/trees/main-root"><strong>root</strong></a> phylogeny (Rep)</li>
			<li> <a href="https://github.com/giffordlabcvr/HHV6-GLUE/tree/master/trees/eve-genus"><strong>Genus-level</strong></a> phylogenies</li>
			<li> <a href="https://github.com/giffordlabcvr/HHV6-GLUE/tree/master/trees/eve-orthologs"><strong>ciHHV6 lineage-level</strong></a> phylogenies</li>
		  </ol>

 		</p>


        
	   <br>
	   <h3>
		 <a id="schemaExtensions" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Paleovirus-specific schema extensions in HHV6-GLUE</strong>
	   </h3>
	   <hr>


		 <p>
		   The paleovirus component of HHV6-GLUE extends GLUE's
		   <a target="_blank" href="http://glue-tools.cvr.gla.ac.uk/#/coreSchema"><b>core schema</b></a>
		   to allow the capture of EVE-specific data.
		   These schema extensions are defined in 
           <a target="_blank" href="https://github.com/giffordlabcvr/HHV6-GLUE/blob/master/glue/eve/ehbvSchemaExtensions.glue"><b>this file</b></a>
		   and comprise two additional tables: 'locus_data' and 'refcon_data'.
		   Both tables are linked to the main 'sequence' table via the 'sequenceID' field.
  		</p>


 		<p>
		   The '<b>locus_data</b>' table contains information pertaining to individual
		   EVE sequences: e.g. species in which they occur, genome assembly version, 
		   genomic location(i.e. scaffold, location coordinates, and orientation).
		</p>

 		<p>
		   The '<b>refcon_data</b>' table contains information pertaining to our
		   ciHHV6 reference sequences, which we have constructed in an effort to
		   reconstruct, as closely as possible, the sequences of the progenitor viruses
		   that gave rise to EVEs. 
		</p>




	   <br>
	   <h3>
		 <a id="Publications" class="anchor" href="#Publications" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Related Publications</strong>
	   </h3>
	   <hr>


		<p>
		
		
		Lytras S, Arriagada G, and RJ Gifford
		<b>(2020)</b>
		<br>
		Ancient evolution of herpesviral paleoviruses and their impact on host genomes.
		<br>
		<i>Virus Evolution</i>
		<a href="https://academic.oup.com/ve/advance-article/doi/10.1093/ve/veab012/6157734" target="_blank">[view]</a>
		<br>
		<br>

		Singer JB, Thomson EC, McLauchlan J, Hughes J, and RJ Gifford
		<b>(2018)</b>
		<br>
		GLUE: A flexible software system for virus sequence data.
		<br>
		<i>BMC Bioinformatics</i>
		<a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2459-9" target="_blank">[view]</a> 
		<br>
		<br>

		Zhu H, Dennis T, Hughes J, and RJ Gifford
		<b>(2018)</b>
		<br>
		Database-integrated genome screening (DIGS): exploring genomes heuristically using sequence similarity search tools and a relational database.
		<a href="https://www.biorxiv.org/content/early/2018/04/25/246835" target="_blank">[preprint]</a> 
		<br>
		<br>

		Gifford RJ, Blomberg B, Coffin JM, Fan H, Heidmann T, Mayer J, Stoye J, Tristem M, and WE Johnson
		<b>(2018)</b>
		<br>
		Nomenclature for endogenous retrovirus (ERV) loci.
		<br>
		<i>Retrovirology</i>
		<a href="https://retrovirology.biomedcentral.com/articles/10.1186/s12977-018-0442-1" target="_blank">[view]</a> 
		<br>
		<br>

		Katzourakis A. and RJ. Gifford
		<b>(2010)</b>
		<br>
		Endogenous viral elements in animal genomes.
		<br>
		<i>PLoS Genetics</i>
		<a href="http://www.plosgenetics.org/article/info:doi/10.1371/journal.pgen.1001191" target="_blank">[view]</a>

 		</p>


	 <footer class="site-footer">
	   <span class="site-footer-owner"><a href="https://github.com/giffordlabcvr/HHV6-GLUE/">HHV6-GLUE</a> is maintained by <a href="https://github.com/giffordlabcvr">giffordlabcvr</a>.</span>

	   <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
	 </footer>

    </section>


  </body>
  
</html>



